### ISSUE TITLE
Unexpected Generated Code for Responses with Status Code 201

### ISSUE THREAD

**Issue by gurumeditator** [atomicbits/scraml#60]
Given the following partial RAML:
```
  responses:
    201:
      description: Authentication success
      body:
        application/json:
          example: !include ../examples/items/token.json
          schema:  !include ../schemas/items/token.json
```

produces a code generated response type of String. However, changing only the 201 status code to a 200 produces the expected response specified in the schema

```
  responses:
    200:
      description: Authentication success
      body:
        application/json:
          example: !include ../examples/items/token.json
          schema:  !include ../schemas/items/token.json
```

**Comment by rigolepe** [comment]
I will fix this one this week. There are plans to support different types for all http status codes. Now, only the 200 status gets typed bodies. I will extend it to accept everything in the 200-range as an OK response with a typed body. I put an issue for status code specific typed responses further down the backlog.

**Comment by gurumeditator** [comment]
Great news. Thanks Peter!

**Comment by rigolepe** [comment]
I released 0.4.12 with a fix for this issue. I also fixed an important bug that prevented proper response handling for responses with status code >= 300.
